flow/legacy.py

```diff
@@ -43,10 +43,10 @@ def wrapper(self, *args, **kwargs):
         # Raise an exception or warn based on the detected API version.
         if api_version is None:
             raise RuntimeError("Unable to determine legacy API use.")
-        elif api_version < 3:
-            raise RuntimeError(
+        elif api_version < 4:
+            warnings.warn(
                 "This FlowProject implementation uses deprecated API "
-                "version 0.{}.x Please downgrade your signac-flow installation "
+                "version 0.{}.x. Please downgrade your signac-flow installation "
```

Comment:
Are these APIs labeled by the _earliest_ version to support that syntax, or the _last_ version to support that syntax? I would rather see the _last_ version somewhere, so that users would know what to install to get maximal features/bugfixes while retaining compatibility.

Reply:
I'm not sure that I understand the question. The APIs are labeled by the version that they were current. I think I am actually going to elevate this warning to an error.